### PR TITLE
[feature/cumulative--10.0.0] AB#33500 - Closed Button Gap

### PR DESCRIPTION
**Pull request by morethanfire**

### CODE REVIEW

File: src/inputs/button/buttonConstants.js [hunk 1]
```diff
@@ -56,6 +61,10 @@ export const buttonPropTypes = {
      */
     style: PropTypes.shape({}),
     target: PropTypes.oneOf(['_blank']),
+    /**
+     * If `true`, the button's text is only showed.
```
Comment (groberts314) [inline src/inputs/button/buttonConstants.js:65]: Maybe: If \`true\`, only the button's text is shown.

**Review by groberts314 — CHANGES_REQUESTED** [review]
Looks pretty good.  I just saw one comment that could maybe be reworded a bit.

File: src/inputs/button/buttonConstants.js (hunk 1 above)
Comment (morethanfire) [reply src/inputs/button/buttonConstants.js:65]: Thanks!

Comment (morethanfire) [reply src/inputs/button/buttonConstants.js]: Fixed.

Comment (groberts314) [reply src/inputs/button/buttonConstants.js]: Lookin' good.  Thank you sir!  Resolving.

**Review by groberts314 — APPROVED** [review]

**Pull request merged**